tests/components/advantage_air/snapshots/test_switch.ambr

```diff
@@ -0,0 +1,43 @@
+# serializer version: 1
+# name: test_cover_async_setup_entry[switch.myzone_myfan-turnoff]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'device_class': 'switch',
+      'friendly_name': 'myzone MyFan',
+      'icon': 'mdi:fan-auto',
+    }),
+    'context': <ANY>,
+    'entity_id': 'switch.myzone_myfan',
+    'last_changed': <ANY>,
+    'last_updated': <ANY>,
+    'state': 'off',
+  })
+# ---
+# name: test_cover_async_setup_entry[switch.myzone_myfan-turnon]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'device_class': 'switch',
+      'friendly_name': 'myzone MyFan',
+      'icon': 'mdi:fan-auto',
+    }),
+    'context': <ANY>,
+    'entity_id': 'switch.myzone_myfan',
+    'last_changed': <ANY>,
+    'last_updated': <ANY>,
+    'state': 'off',
```

Comment:
Shouldn't `turn_on` actually turn it on?

Reply:
Ah yes, snapshot testing is no appropriate here. Ill change it to check the mock arguments.

Reply:
Ok snapshot testing is fine. State testing isnt. Testing arguments with snapshots now.